11/chapter-12.srt

```diff
@@ -332,8 +332,8 @@ the mtime of that file at that point.
 
 73
 00:07:31,051 --> 00:07:35,622
-But your talking about almost the end
-point of that file after it's been regenerated.
+But your talking about almost the mtime
```

Comment:
this should also be you're not your! I missed it... can you change it your end? Much easier.